### DISCUSSION TITLE
What's the lastest version for Ubuntu 23.04?

### DISCUSSION THREAD

**Discussion by ThaDaVos** [cockpit-project/cockpit#20713] [Q&A]
My current install is Cockpit 308 - but I see there's already a 320 - but `apt-get` doesn't show any updates for Cockpit, even with `lunar-backports` enabled as suggested

**Comment by martinpitt** [comment]
None at all -- non-LTS Ubuntu versions are just supported for 9 months, and 23.04 is end of life. Please upgrade to 24.04 LTS.
